src/Customer.php

```diff
@@ -110,9 +108,20 @@ public function notes()
     public function lineItems()
     {
         $line = new LineItem($this->_client);
-        $line->setEndpointBase($this->getEndpoint());
 
-        return $line;
+        return $line->setEndpointBase($this->getEndpoint());
+    }
+
+    /**
+     * Gets a payment source object for this customer.
+     *
+     * @return PaymentSource
+     */
+    public function paymentSources()
+    {
+        $source = new PaymentSource($this->_client);
+
+        return $source->setEndpointBase($this->getEndpoint());
```

Comment:
If you are going to do the fluent setter return shortcut can you extend that to the rest of the application?

Reply:
Ah sure. I did it throughout Customer. Will do now on Invoice

Reply:
ok all set